### ISSUE TITLE
DFU Serial Instructions

### ISSUE THREAD

**Issue by boppana113** [sparkfun/nRF52832_Breakout#3]
Hi, I am using the Sparfun NRF52 breakout for a project and I want to develop my own PCB with it later. Is there any simple instruction guide on how to burn this specific bootloader program? Is there any specific hardware or software needed?

Thank you.

**Comment by jimblom** [comment]
I've cleaned up the Makefile, and added a bit more to the [bootloader README](https://github.com/sparkfun/nRF52832_Breakout/blob/master/README.md), which documents how to build the bootloader.

To burn the bootloader onto an nRF52832, you'll need a hardware programmer. I'd recommend grabbing an [nRF52832 Development Kit](http://infocenter.nordicsemi.com/topic/com.nordic.infocenter.nrf52/dita/nrf52/development/nrf52_dev_kit.html?cp=2_2_2), which can serve as a development platform for the nRF52 as well as a programmer for external chips.

Software tools like [nRFgo Studio](https://www.nordicsemi.com/eng/Products/2.4GHz-RF/nRFgo-Studio/) or [Nordic's command-line tools](http://infocenter.nordicsemi.com/topic/com.nordic.infocenter.tools/dita/tools/nrf5x_command_line_tools/nrf5x_command_line_tools_lpage.html) can be used to load code onto the chip.

**Comment by boppana113** [comment]
Hi Jimblom. I got the NRF52 DK and the Jlink EDU programmer. I was able to sucessfully program the sparkfun NRF52832 breakout board using Keil (J-link EDU). However, now I can't seem to go back and program through arduino. When I try to burn the bootloader using NRFgo studio and the bootloader hex file "sfe_nrf52832_dfu.hex" found in the repository it fails and gives an output saying:

![image](https://cloud.githubusercontent.com/assets/19865684/24568653/cd089808-1630-11e7-83ee-ef6d7027387f.png)

Im not sure how to go around this. Thanks.
